src/osd/OpRequest.cc

```diff
@@ -93,6 +93,14 @@ void OpRequest::_dump_op_descriptor_unlocked(ostream& stream) const
   get_req()->print(stream);
 }
 
+void OpRequest::_fetch_poolid(uint64_t& poolid)
```

Comment:
why this format?
why not using a member function that returns the poolid?
If the issue is what happens when we are not dealing with FastDispatcher - the current signature does not help,
as the pool-id is silently left unchanged.